components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -526,7 +527,7 @@ export default class AdvancedSettingsDisplay extends React.PureComponent {
                         describe={
                             <FormattedMessage
                                 id='user.settings.advance.enabledFeatures'
-                                defaultMessage='{count, number} {count, plural, one {Feature} other {Features}} Enabled'
+                                defaultMessage='{count, number} {count, plural, one {feature} other {features}} enabled'
```

Comment:
This is just to make the description text in the pre-release features setting consistently cased with other settings since it was bugging me that it was different